azext_iot/central/commands_scheduled_job.py

```diff
@@ -0,0 +1,185 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# Dev note - think of this as a controller
+
+from typing import List
+
+from azure.cli.core.azclierror import InvalidArgumentValueError
+from azext_iot.constants import CENTRAL_ENDPOINT
+from azext_iot.central.providers import CentralScheduledJobProvider
+from azext_iot.common import utility
+from azext_iot.central.common import API_VERSION
+from azext_iot.central.models.ga_2022_07_31 import ScheduledJobGa
+
+
+def get_scheduled_job(
+    cmd,
+    app_id: str,
+    job_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=API_VERSION,
+) -> ScheduledJobGa:
+    provider = CentralScheduledJobProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    return provider.get_scheduled_job(
+        job_id=job_id,
+        central_dns_suffix=central_dns_suffix,
+    )
+
+
+def list_scheduled_jobs(
+    cmd,
+    app_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=API_VERSION,
+) -> List[ScheduledJobGa]:
+    provider = CentralScheduledJobProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+
+    return provider.list_scheduled_jobs(central_dns_suffix=central_dns_suffix)
+
+
+def create_scheduled_job(
+    cmd,
+    app_id: str,
+    job_id: str,
+    group_id: str,
+    schedule: str,
+    content: str,
+    job_name=None,
+    description=None,
+    batch_type=None,
+    threshold_type=None,
+    threshold_batch=None,
+    batch=None,
+    threshold=None,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=API_VERSION,
+) -> ScheduledJobGa:
+    if not isinstance(content, str):
+        raise InvalidArgumentValueError("content must be a string: {}".format(content))
```

Comment:
I am pretty sure that the az cli takes care of this - almost everything is a string (except if you type it as not a str)

Reply:
Right, I just copied it from some old codes. Let me remove this.